libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -79,6 +103,13 @@ bool DocStringAnalyser::visit(VariableDeclaration const& _variable)
 				"Documentation tag @title and @author is only allowed on contract definitions. "
 				"It will be disallowed in 0.7.0."
 			);
+
+	if (
+		_variable.annotation().docTags.empty() &&
+		_variable.annotation().baseFunctions.size() == 1
+	)
+		copyMissingTags(_variable.annotation(), (*_variable.annotation().baseFunctions.begin())->annotation());
```

Comment:
If you `dynamic_cast` inside `copyMissingTags` anyway, you could also move the `->annotation()` call inside `copyMissingTags`. Maybe even make the `baseFunctions` the parameter and then you can also move the check for `baseFunctions.size() == 1`

Reply:
I can do that only partially. I still need to do the `.size() == 1` check for the callable function as we also check the name&parameter types first.